client/js/components/month/month-eventcontainer.js

```diff
@@ -1,18 +1,18 @@
 /** @jsx React.DOM */
 var React = require('react');
-var Event = require('./month-event.js');
+var MonthWeek = require('./month-week-board.js');
+var _  = require('underscore');
 
 var MonthEventContainer = React.createClass({
   render:function() {
+    var monthWeeks = _.map(this.props.monthEvents, function(week, key) {
```

Comment:
If you are mapping over the property monthEvents, you'd probably expect that each item would be an event. It seems, based on your example data, like the property would be better described as this.props.monthWeeks. That way, each item in monthWeeks corresponds to a single week.